src/styles/quiz-card/quiz-card.html

```diff
@@ -0,0 +1,15 @@
+<section class="quiz-card">
+  <h1 class="quiz-card__headline">Questions</h1>
+  <div role="button" class="quiz-card__bookmark"></div>
```

Comment:
I'd also add an aria-label
```suggestion
  <div role="button" class="quiz-card__bookmark" aria-label="Add bookmark"></div>
```